addon/data/shoot-panel.js

```diff
@@ -59,20 +65,24 @@ let ShootPanel = React.createClass({
         <a href="#">+</a>
       </div>
       <div className="link-row">
-        <a className="link" target="_blank" href={ this.props.viewUrl } onClick={ this.onLinkClick }>{ this.props.viewUrl }</a>
+        <a className="link" target="_blank" href={ this.props.shot.viewUrl } onClick={ this.onLinkClick }>{ this.props.shot.viewUrl }</a>
         <button className="copy" ref="copy" type="button" data-normal-text="Copy Link" data-copied-text="Copied!" onClick={ this.onCopyClick }>Copy Link</button>
       </div>
 
 
-      <div className="text">{ this.props.textSelection }</div>
-      <div className="comment">{ this.props.comment}</div>
+      <div className="text">{ this.props.shot.textSelection }</div>
```

Comment:
There's no textSelection property, what is this supposed to be?  Maybe the text selection stuff that was in there before?  That was never on the shot properly, just a kind of little test to surface that selection functionality.  Text selections will be clips in the future (and the model supports that).